### ISSUE TITLE
CSLA.NET 4.6.300 with ASP.NET 5 and MVC 6

### ISSUE THREAD

**Issue by ConstantinPatak** [MarimerLLC/cslaforum#162]
Hello everybody,

We need to create an API for our CSLA BLL and we are thinking to use MVC 6. Is CSLA fully compatible with MVC 6? Does anyone have any experience with this?

As I read CSLA 4.6.300 provide support for ASP.NET 5 does that mean that we can use MVC 6 with CSLA?

Thanks in advance,
Constantin Patak

**Comment by jonnybee** [comment]
I assume your BLL does not reference MVC assemblies? 

Csla.Mvc is basically a few classes for controller, modelbinder and other helper classes. 

You could create your own implementation that references MVC 6 -  or just bake sure that you BLL does not reference any MVC assemblies.

**Comment by mvklingeren** [comment]
I agree with jonnybee, your BLL should not be bound to Microsoft...Asp.Mvc...
using CSLA should be not be the problem.
